### DISCUSSION TITLE
Trying to get back to AIR

### DISCUSSION THREAD

**Discussion by cptvideo** [airsdk/Adobe-Runtime-Support#1230] [General]
I’m an app developer hoping to come back to AIR after a year off.  A client of mine needs me to continue to develop their iOS and Android apps, ideally taking advantage of the economics of my old tool-kit: AIR, Animate, Flash-Develop. So,a few questions:

Is it still feasible to develop for iOS and Android, more-or-less simultaneously on Windows? 

Can iOS Enterprise apps that were working properly on iOS 14 be simply re-signed in order to run on iOS 15? In essentially the same way that they had to be re-signed to jump from iOS13 to iOS 14?

Is AIR 33.1.1.633 compatible with iOS 15 right out of the ‘box’ for iOS Enterprise distribution?

Do we packaging ipas and apks just like we always always did with Flash/Animate?

And, is this site the current best place to discuss things like workflow, and the nitty-gritty of AS3 coding with the current version of AIR.

**Comment by ajwfrost** [comment]
I'm hoping that "yes" should cover pretty much all of those questions!

The re-signing thing would need to be done on a mac, and there may also need to be some tweaks to the Info.plist values so that the IPA file shows up as being built by the latest Xcode/SDK (although I think that's more a requirement for App Store deployments).

33.1.1.633 works for most iOS15 deployments, the one thing we have found out is an issue with signing dynamic framework libraries (i.e. we're signing them with entitlements but we shouldn't be ...) but as long as you don't have that, it should be okay.

If you're uploading to the Android Play Store then you'll need to package as an App Bundle which needs a bit of a workaround when using Animate since there's no UI to select this yet.

And yes this is definitely the best place to discuss things... there are a lot of folk also using facebook, too many different groups are active there though imho! so it would be nice to keep everything in one place.

thanks

**Reply by dgurney75** [reply]
I've started working on a base application that uses Adobe Animate CC as the dev platform. Right now, I just have a login/create 
new account user flow working with a sample UI. I'm hoping to build more UI that supports common mobile activities. (ex. - Complex scrolling lists, carousels, image editors, etc. ) It would require the District ANEs to publish, but i think it could be very powerful when complete. Anyway, here's the link to the repository in Github if interested.

https://github.com/dgurney75/Air-Base-App

**Reply by marchbold** [reply]
@dgurney75 Might be useful to create a apm project in that repo so users can just do an `apm install` and get the extensions (if they are available)? Let me know if you need some guidance in doing this.
